src/.vuepress/config.js

```diff
@@ -41,52 +41,90 @@ module.exports = {
     displayAllHeaders: true,
     docsRepo: 'CudoVentures/cudos-network-docs',
     docsDir: 'src',
-    editLinks: true,
+    // editLinks: true,
     sidebarDepth: 1,
-    nav: [
-      { text: 'Learn', link: '/learn/' },
-      { text: 'Build', link: '/build/' },
-      { text: 'Earn', link: '/earn/' }
-    ],
+    //    nav: [
+    //      { text: 'Learn', link: '/learn/' },
+    //      { text: 'Build', link: '/build/' },
+    //      { text: 'Earn', link: '/earn/' }
+    //    ],
     logo: '/CudosIconTransparent.png',
     repo: 'CudoVentures/',
-    sidebar: {
-      '/learn/': [
-        '',
-        'roadmap',
-        'validators',
-        'gravity-bridge',
-        'resources'
-      ],
-
-      '/build/': [
-        '',
-        'prerequisites',
-        'account-setup',
-        'funding-your-wallet',
-        'validator',
-        'validator-restart',
-        'developers-setup',
-        'smart-contracts',
-        'nft',
-        'interact-keplr-explorer',
-        'upgrade',
-        'upgrade-v0.4',
-        'start-binaries'
-      ],
-
-      '/earn/': [
-        '',
-        'staking',
-        'incentives',
-        'grants'
-      ],
-
-      // fallback
-      '/': [
-        '',
-      ]
-    }
+    sidebar: [
+      {
+        title: 'Learn',
+        collapsible: true,
+        children: ['/learn/cudosnetwork.md', '/earn/staking.md', '/learn/validators.md', '/learn/roadmap.md'],
+      },
 
+      {
+        title: 'Build',
+        collapsible: true,
+        children: ['/build/tbcplaceholder.md', '/build/hardware.md'],
+      },
+      {
+        title: 'Funding & Earning with CUDOS Tokens',
+        collapsible: true,
+        children: ['/build/tbcplaceholder.md'],
+      },
+      {
+        title: 'Developer Guides',
+        collapsible: true,
+        children: ['/build/tbcplaceholder.md'],
+      },
+      {
+        title: 'Operational Tasks & Troubleshooting',
+        collapsible: true,
+        children: ['/build/tbcplaceholder.md'],
+      },
+      {
+        title: 'Upgrade',
+        collapsible: true,
+        children: ['/build/upgrade.md'],
+      },
+      {
+        title: 'Testnet',
+        collapsible: true,
+        children: ['/testnet/testnetoverview.md', '/testnet/testnetenvprep.md', '/testnet/testnetcluster.md', '/testnet/testnetstandalone.md', '/build/securingnodes.md',
+          {
+            title: 'Testnet Hardware Requirements',
+            collapsible: true,
+            children: ['/testnet/testnet-hardware.md'],
+          },
+          {
+            title: 'Funding and earning with CUDOS tokens on Testnet',
+            collapsible: true,
+            children: ['/build/fundnodes.md', '/build/interact-keplr-explorer.md', '/build/proposals.md']
```

Comment:
I think the 'Common tasks with Keplr wallet and Cudos Explorer' section (aka `/build/interact-keplr-explorer.md`) could be extracted from this nested list to it's own higher level section. Similarly with 'Governance Proposals'. These are almost better described as User Guides (rather than Developer or Validator Guides)